python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -3705,6 +3705,22 @@ keys:
                     description: IPv4 prefix "A.B.C.D/E" or IPv6 prefix "A:B:C:D:E:F:G:H/I".
                   route_map:
                     type: str
+            maximum_paths:
+              type: dict
+              keys:
+                paths:
+                  type: int
+                  required: true
+                  convert_types:
+                    - str
+                  min: 1
+                  max: 600
```

Comment:
Can you please confirm about EOS version on which you got the limit of paths and ecmp.
Because in mine one i see the limit 1 to 128

Reply:
Hi Mahesh,

I can see this limits in 4.28.3M, please see below:

```
router(config-router-bgp-vrf-168)#sh version | i Software
Software image version: 4.28.3M
router(config-router-bgp)#   vrf 168
router(config-router-bgp-vrf-168)#maximum-paths ?
  <1-600>  Value for maximum number of equal cost paths

router(config-router-bgp-vrf-168)#maximum-paths 1 ?
  ecmp  Maximum number of installed ECMP routes
  <cr>  

router(config-router-bgp-vrf-168)#maximum-paths 1 ecmp ?
  <1-600>  Value for maximum number of installed ECMP routes

```

Reply:
Thanks for the output. Got to know that range its platform dependent.